lib/extensions.js

```diff
@@ -16,8 +16,14 @@ class Extensions extends ApiGroup {
     options = Object.assign({}, options, {
       path: 'apis/extensions',
       version: options.version || 'v1beta1',
-      groupResources: resources,
-      namespaceResources: resources
+      groupResources: commonResources.concat(['deployments']),
+      //
+      // The custom Deployments objects implement functionality available only
+      // in namespaces.
+      //
+      namespaceResources: commonResources.concat([
```

Comment:
Is this property used anywhere? I did a ctrl+f through GitHub and didn't see any references to it outside of assignments.